crates/bevy_core_pipeline/src/core_2d/mod.rs

```diff
@@ -149,13 +150,15 @@ impl CachedRenderPipelinePhaseItem for Transparent2d {
 
 pub fn extract_core_2d_camera_phases(
     mut commands: Commands,
+    mut transparent_2d_phases: ResMut<ViewSortedRenderPhases<Transparent2d>>,
     cameras_2d: Extract<Query<(Entity, &Camera), With<Camera2d>>>,
 ) {
     for (entity, camera) in &cameras_2d {
-        if camera.is_active {
-            commands
-                .get_or_spawn(entity)
-                .insert(SortedRenderPhase::<Transparent2d>::default());
+        if !camera.is_active {
+            continue;
         }
+
+        commands.get_or_spawn(entity);
+        transparent_2d_phases.insert_or_clear(entity);
```

Comment:
if users spawn/destroy many views (something i do for worldspace ui, for example) i guess this may get slower as the hashmap fills up. could we retain only the used views?

Reply:
Done.